### PR TITLE
Mars 1.50 - Patch 1

### PR DESCRIPTION
**Pull request by RedFox20**
Audio: added missing sound effects
Audio: fixed minor bugs and issues in the new audio engine
Audio: preview Effect volume in Options screen and fix a critical Async play bug with AudioHandles
Audio: reduce audio buffer duration and disable event sync
fix: improve projector search performance
Fix: allow changing music volume inside universe screen
Fix: IconSize must be at least 1
Fix: Adjust  warp exit space.
Fix: Player ship commands movement limitation
Fix: Added missing weapons audio
Fix: Check Mining Ops ownership for starting systems. 
Fix: UI for unexplored planet screen.
Fix: ability to add mining stations above limit.
Fix: Encapsulate and cache ShipCostMultiplier to avoid concurrency issues with NaN.
Fix: Update shipyards when tether is removed or added.
Refactor: Research screen hardcoded text to localization
Balance: Change FindShipAt scan from 10K to 150K
Balance: Remnant Portal defense logic.
Refactor: Remove unused parameter in AffectNearbyShips. 
Refactor: Added NaN trace for repair.
@RedFox20 , @gkapulis, @InvyX , @SmischenkoB

### CODE REVIEW

**Review by gkapulis — APPROVED** [review]

File: Ship_Game/Audio/AudioConfig.cs [hunk 1]
```diff
@@ -222,7 +224,7 @@ public void Update()
         // remove disposed handles
         lock (TrackedInstances)
         {
-            for (int i = 0; i < TrackedInstances.Count; i++)
+            for (int i = 0; i < TrackedInstances.Count; ++i)
```
Comment (SmischenkoB) [inline Ship_Game/Audio/AudioConfig.cs:227]: Is there any purpose of ++i? Or it can be TrackedInstances.Count + 1?

Comment (gkapulis) [reply Ship_Game/Audio/AudioConfig.cs:227]: Its a shortcut for  `i++`
also `++i` and `i++` differ

Comment (RedFox20) [reply Ship_Game/Audio/AudioConfig.cs:227]: It's called the idiomatic for loop. You should simply memorize how this is done. It is always the same everywhere.

```cs
for (int i = 0; i < array.Length; ++i)
   Log.Info(array[i].ToString()); 
```

Comment (SmischenkoB) [reply Ship_Game/Audio/AudioConfig.cs:227]: I thought i++ is kinda more usual

Comment (RedFox20) [reply Ship_Game/Audio/AudioConfig.cs:227]: It doesn't make a difference in the big picture.

But if you really care about the semantics
++i means pre-increment
i++ means post-increment

**Pull request merged**